### DISCUSSION TITLE
ShowInMenuBar Minutes Display

### DISCUSSION THREAD

**Discussion by amyers22** [SAP/macOS-enterprise-privileges#158] [Ideas]
With the new `ShowInMenuBar` key in the 2.1.0b1925 beta, it would be cool to also display the remaining mins in the menu bar icon either once under a certain threshold, 5 minutes, or have that as a configurable option in itself.

**Comment by mthielemann** [comment]
If you move the mouse over the status item, the remaining time is shown as a tooltip. Does this work for you?

**Reply by amyers22** [reply]
Ah, yes, after hovering on it for about 2 seconds it shows.
